ttnn/ttnn/operations/core.py

```diff
@@ -201,14 +201,17 @@ def from_torch(
 
     if mesh_mapper:
         shards = mesh_mapper.map(tensor)
-        tensor = ttnn.Tensor(shards, dtype, mesh_mapper.config())
+        if tile is not None:
+            tensor = ttnn.Tensor(shards, dtype, mesh_mapper.config(), tile)
+        else:
+            tensor = ttnn.Tensor(shards, dtype, mesh_mapper.config())
     else:
         if tile is not None:
             tensor = ttnn.Tensor(tensor, dtype, {}, tile)
         else:
             tensor = ttnn.Tensor(tensor, dtype)
 
-    if layout is not None:
+    if layout is not None and not (dtype == ttnn.bfloat8_b or dtype == ttnn.bfloat4_b):
```

Comment:
Please add this behaviour to the docstrings to from_torch/to_torch.

Reply:
added